agent.go

```diff
@@ -787,6 +795,85 @@ func (a *Agent) addRemotePassiveTCPCandidate(remoteCandidate Candidate) {
 		a.localCandidates[localCandidate.NetworkType()] = append(a.localCandidates[localCandidate.NetworkType()], localCandidate)
 		a.candidateNotifier.EnqueueCandidate(localCandidate)
 
+		a.log.Debugf("Adding active tcp host candidate. LocalCandidate: %v RemoteCandidate: %v",
+			localCandidate.addr(), remoteCandidate.addr())
+		a.addPair(localCandidate, remoteCandidate)
```

Comment:
I didn't actually add this line. It's a bad diff. See the bottom of the following "for each stun uri" loop -- has the same `addPair` but is not considered a new line.